### PR TITLE
Fixed responsiveness of modify order page

### PR DESCRIPTION
**Pull request by atulyadav745**
This PR fixes the issue #747  which involves improving overall Viewport Compatibility, Layout Adjustment across all devices of modify order page.  (on max-width <1056px),

Before:
befi

After:
After1

@mozzy11 This PR is ready to review.
Thank you

### CODE REVIEW

File: frontend/src/components/modifyOrder/SearchOrder.js [hunk 1]
```diff
@@ -39,13 +39,10 @@ function SearchOrder() {
     <>
       <div className="orderLegendBody">
         <Form onSubmit={handleSearch}>
-          <Grid>
-            <Column lg={16}>
```
Comment (adityadeshlahre) [inline frontend/src/components/modifyOrder/SearchOrder.js:43]: Hey @atulyadav745, in our codebase, grids and columns are used to maintain consistency across the code. Removing them isn't a good idea as they serve a purpose. We may need to find a better way to fix the responsiveness of the following pages!
Thank You :slightly_smiling_face:

Comment (CalebSLane) [reply frontend/src/components/modifyOrder/SearchOrder.js:43]: I don't have a strong preference one way or the other, but using Grid does allow a nice level of control at different viewport sizes. To fix the viewport issue while still using Grid would require specifying the column span at medium and small viewport sizes (ex. `<Column lg={16} md={8} sm={4}></Column>`) In React Carbon, the grid system is 16 wide at the large size, 8 at the medium, and 4 at the small. Details of Grid are [here](https://react.carbondesignsystem.com/?path=/story/components-ui-shell--header-base)

@mozzy11 do you have any opinions on which strategy should be preferred?

Comment (mozzy11) [reply frontend/src/components/modifyOrder/SearchOrder.js:43]: @CalebSLane  i agree with using the Carbon Grid. it also makes it easier to have  consistence across the project . Innitialy we used custom classes but it creates more issues as compared to using the Carbon Grid

**Review by codewithunknown — APPROVED** [review]
LGTM

**Review by mozzy11 — APPROVED** [review]

**Pull request merged**